weather_mv/loader_pipeline/sinks_test.py

```diff
@@ -38,6 +43,25 @@ def decorated(*args, **kwargs):
     return decorated
 
 
+def generate_dataset() -> str:
+    """Generates temporary netCDF file using xarray."""
+    lat_dim = 3210
+    lon_dim = 3440
+    lat = [0] * lat_dim
+    lon = [0] * lon_dim
```

Comment:
Let's do something like this instead: 
```suggestion
    lat_dim = 3210
    lon_dim = 3440
    lat = np.linspace(-90, 90, lat_dim)
    lon = np.linspace(-180, 180, lon_dim)
```